### PR TITLE
PMM-3478: Change runs_on_node_id to pmm_agent_id

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

File: api/inventory/agents.proto [hunk 1]
```diff
@@ -51,7 +51,7 @@ message MySQLdExporter {
   // Unique randomly generated instance identifier.
   string agent_id = 1;
   // Node identifier where this instance runs.
```
Comment (AlekSi) [inline api/inventory/agents.proto]: Comment should be updated everywhere.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-282/

File: api/inventory/agents.proto [hunk 2]
```diff
@@ -158,8 +158,8 @@ message QANMySQLPerfSchemaAgent {
 
 message ListAgentsRequest {
   oneof filter {
-    // Return only Agents running on that Node.
-    string runs_on_node_id = 1;
+    // Return only Agents running near concrete PMMAgent.
```
Comment (AlekSi) [inline api/inventory/agents.proto]: Not "near". pmm-agent starts other Agents.
And not "concrete". That means "solid" and "physical", but it's not solid or physical. (It does not mean "specific" in the same sense as word "конкретный").

// Return only Agents started by this pmm-agent.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-282/

File: api/inventory/agents.proto [hunk 3]
```diff
@@ -28,8 +28,8 @@ message PMMAgent {
 message NodeExporter {
   // Unique randomly generated instance identifier.
   string agent_id = 1;
-  // Node identifier where this instance runs.
-  string node_id = 2;
+  // pmm-agent identifier where this instance runs.
```
Comment (AlekSi) [inline api/inventory/agents.proto]:
```suggestion
  // The pmm-agent identifier which runs this instance.
```

Do the same everywhere.

Comment (DexterHD) [reply api/inventory/agents.proto]: This is your comment just copied from https://github.com/percona/pmm/blob/0efdbf9d6a2a97301af3849456dacbd0189839be/api/inventory/agents.proto#L151 but nevermind, I'll fix that.

Comment (AlekSi) [reply api/inventory/agents.proto]: Yes, but it makes less sense. :) "where" points to the place. The place stays the same – some Node (host). pmm-agent is not a place – it is a way to run agent.

**Comment by AlekSi** [comment]
For you to merge together with PRs for pmm-managed and (maybe) pmm-agent.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-282/

**Pull request merged**